linkerd/dns/src/lib.rs

```diff
@@ -60,15 +62,12 @@ impl Resolver {
     /// NOTE: It would be nice to be able to return a named type rather than
     ///       `impl Future` for the background future; it would be called
     ///       `Background` or `ResolverBackground` if that were possible.
```

Comment:
I think this comment can be removed now that we have a named return type rather than `impl Future`?